docs/cudf/source/user_guide/guide-to-udfs.ipynb

```diff
@@ -20,6 +20,16 @@
     "import numpy as np"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    strings_enabled = cudf.core.udf._STRING_UDFS_ENABLED
</pre><p> </p><p>- The variable name strings_enabled feels misleading (it indicates general support for strings, rather than just for UDFs)</p><p> </p><p>- Can we move this closer to where it's actually used? It's fine if we compute it redundantly for both the <code>Series</code> and <code>DataFrame</code> sections.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/11861/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>